src/com/simulator/model/Zone.java

```diff
@@ -0,0 +1,115 @@
+package com.simulator.model;
+
+import com.simulator.controller.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Zone {
+    private List<Room> roomArrayList = new ArrayList<Room>();
+    private String zoneName;
+    private int temperatureMorning;
+    private int temperatureDay;
+    private int temperatureNight;
+    private boolean emptyZone;
+
+    public Zone(ArrayList<Room> rl, String zN){
+        this.roomArrayList = rl;
+        this.zoneName = zN;
+    }
+    //if temperature has not been overwritten, update the temperature for the morning, daytime and nighttime
```

Comment:
What is the significance of a room (comment says temperature, but I think meant room) being "overwritten"?